### ISSUE TITLE
`pkgs.gnome.gnome-backgrounds` is now top level package `pkgs.gnome-backgrounds`

### ISSUE THREAD

**Issue by knoopx** [danth/stylix#528]
looks like upstream changes are breaking stylix:

https://github.com/NixOS/nixpkgs/commit/6e8760f7f7121128e2037db44915a4a5450b6e67

```
       error: attribute 'gnome-backgrounds' missing

       at /nix/store/qb1mjpc9gh2w20wmrlf38v1y23nnzadz-modules/gnome/nixos.nix:22:43:

           21|     # which will then download the pack regardless of its exclusion below.
           22|     environment.gnome.excludePackages = [ pkgs.gnome.gnome-backgrounds ];
```

**Comment by r00tr4v3n** [comment]
How can this be temporarily fixed for 24.05/unstable?

**Comment by trueNAHO** [comment]
> How can this be temporarily fixed for 24.05/unstable?

Do you mean the https://github.com/danth/stylix/tree/release-24.05 branch? Maybe we should backport this patch.

**Comment by brumik** [comment]
I do not think this plays together with 24.05, I am on nixpgs 24.05 and master on stylix got broken for me after updating the flakes. I had to go to `stylix/release-24.05` to get my system to build again.
